app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/VpnFeaturesRegistryImpl.kt

```diff
@@ -44,6 +47,7 @@ internal class VpnFeaturesRegistryImpl(
             putString(feature.featureName, UUID.randomUUID().toString())
         }
         vpnServiceWrapper.restartVpnService(forceRestart = true)
+        featureSegmentsManager.addUserToFeatureSegment(FeatureSegmentType.APP_TP_ENABLED)
```

Comment:
@nalcalag I don't think this is the way to proceed. I know this is temporary code, but it's cluttering a lot of places we don't want it to clutter that will make difficult its removal later on